websites/A/AnimesWatch/presence.ts

```diff
@@ -0,0 +1,80 @@
+const presence = new Presence({
+		clientId: "972647321970016316",
+	}),
+	getStrings = async () =>
+		presence.getStrings(
+			{
+				playing: "general.playing",
+				paused: "general.paused",
+			},
+			await presence.getSetting<string>("lang").catch(() => "tr")
+		),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video: HTMLVideoElement, strings: Awaited<ReturnType<typeof getStrings>>;
+
+presence.on("iFrameData", async (msg: HTMLVideoElement) => {
+	if (!msg) return;
+	video = msg;
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "animeswatch",
+			details: "Ana Sayfada Göz Gezdiriyor",
+		},
+		{ pathname } = document.location,
+		[button, timestamp] = await Promise.all([
+			presence.getSetting<boolean>("button"),
+			presence.getSetting<boolean>("timestamp"),
+		]),
+		title = document.title.slice(0, document.title.length - 14);
```

Comment:
You can assign the value here instead of repeating it

```suggestion
		presenceData.state = document.title.slice(0, document.title.length - 14);
```